lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -198,6 +198,7 @@ def issued_certificates(course_key, features):
 
     # Report run date
     for data in generated_certificates:
+        data['course_id'] = str(data['course_id'])
```

Comment:
In other places we're using `unicode()` to deal with these, I suspect this one is on me but just to confirm these should be getting stringified via `unicode` and not `str`?

Reply:
Yeah - `unicode` is the way to go.

Reply:
Which is not Python3 friendly, since there's no more `unicode()` in Python3 - but, another day...